lib/mbta.js

```diff
@@ -1,10 +1,22 @@
 'use strict'
 
-// Code here.
-
-const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
+const red = ['South Station', 'Park Street', 'Kendall', 'Central', 'Harvard',
```

Comment:
Rather than creating a bunch of constants, the preferable way to represent your stations is to have them all in one object: `const subwayLines = {
  Red: [
    'South Station',
    'Park Street',
    'Kendall',
    'Central',
    'Harvard',
    'Porter',
    'Davis',
    'Alewife'
  ],
  Green: [
    'Government Center',
    'Park Street',
    'Boylston',
    'Arlington',
    'Copley',
    'Hynes',
    'Kenmore'
  ],
  Orange: [
    'North Station',
    'Haymarket',
    'Park Street',
    'State',
    'Downtown Crossing',
    'Chinatown',
    'Back Bay',
    'Forest Hills'
  ]
}`